style/checkstyle.xml

```diff
@@ -34,7 +34,7 @@
 <module name = "Checker">
     <property name="charset" value="UTF-8"/>
 
-    <property name="severity" value="warning"/>
+    <property name="severity" value="error"/>
```

Comment:
If we change this level, whether it would break the build progress for those modules which do not refactor the import orders currently?

Reply:
No impact

Reply:
OK